packages/rolldown/package.json

```diff
@@ -65,7 +65,17 @@
     "# Scrips for checking #": "_",
     "test": "vitest run --reporter verbose --hideSkippedTests",
     "test:update": "vitest run -u",
-    "type-check": "tsc"
+    "type-check": "tsc",
+    "circular-dep-check": "madge --circular src/index.ts src/cli/index.ts src/parallel-plugin.ts src/parallel-plugin-worker.ts"
```

Comment:
I am not sure if we should introduce another tool to do this, @hyf0

Reply:
Not needed. We have rolldown already as the check tools.